src/core/browser/qgsdataitemproviderregistry.cpp

```diff
@@ -25,6 +25,12 @@ QgsDataItemProviderRegistry::QgsDataItemProviderRegistry()
 {
   QStringList providersList = QgsProviderRegistry::instance()->providerList();
 
+  // OWS is not useful in the browser.
+  // The OWS type connections (WFS/WMS) are available in their respective menus.
+  // Having an OWS menu in the browser only brings disadvantages.
+  // See https://github.com/qgis/QGIS/issues/24841
+  providersList.removeOne( QStringLiteral( "ows" ) );
```

Comment:
This is an odd way to remove it -- it's still being created and then deleted. Better would be to just remove QgsOwsProviderMetadata::dataItemProviders() and qgsowsdataitem.h/.cpp entirely.

Reply:
Actually just remove the whole src/providers/ows folder, none of that is required

Reply:
Indeed.

note: I spent a few minutes wondering why ows was still appearing in the browser even though I had deleted its code... libowsprovider.so was still in my build folder and therefore remained present. :sweat_smile: